guide/10-mapping-and-visualization/usage-of-visualVariables-and-other-renderer-options-in-map-widget.ipynb

```diff
@@ -12,7 +12,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can we create a FeatureLayer object that points to this URL and add it in this snippet and allow other snippets that use this layer as</p><p><br></p><p>```</p><p>usa_flayer = FeatureLayer(url="...")</p><p>`map1.add_layer(usa_flayer, options={"renderer":"...","field_name":"...")</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>